nav2_collision_monitor/README.md

```diff
@@ -0,0 +1,61 @@
+# Nav2 Collision Monitor
+
+Collision Monitor - is an independent layer in Nav2 providing an additional level of robot safety assurance.
+It allows performing robot collision avoidance by monitoring the obstacles in surrounding environments laying in collision proximity to the robot.
+
+This is extremely useful and integral part of large heavy industrial robots operating in the facilities, or of robots moving with high velocities, where the collision with any obstacle might cause unpredictable severe consequences.
+
+## Features
+
+Collision Monitor operates by means of areas exposed around the robot and moving with it.
+The obstacles fell into such area are forcing robot to stop, operate slowly or even fluently approach an obstacle with a constant time to the collision.
+
+The following models of safety behaviors are employed by Collision Monitor:
+
+* **Stop model**: Define a safety area surrounding the robot and a point threshold. If more that `N` obstacle points appear inside this area, stop the robot until the obstacles will disappear.
+* **Slowdown model**: Define a safety area around the robot and slow the maximum speed for a `%S` percent, if more than `N` points will appear inside the area.
+* **Approach model**: With the current robot speed, estimate the time to collision to points obtained from sensors. If the time is less than `M` seconds (0.5, 2, 5, etc...), slow the robot until it will be equal to `M` seconds. The effect here would be to keep the robot always `M` seconds from a collision and continuously scale down its speeds.
+
+The safety area around the robot can take the following shapes:
+
+* Arbitrary user-defined polygon around the robot for usage in stop and slowdown models.
+* Robot footprint polygon, which is used in the approach behavior model only.
+* Circle, used in all models: for stop and slowdown models as a safety area, for approach model as a robot footprint. Circle is made for the best performance and could be used in the cases where the safety area or robot could be approximated by round shape.
```

Comment:
```suggestion
* Circle: is made for the best performance and could be used in the cases where the safety area or robot could be approximated by round shape.
```

Move circle before the footprint so the general ones are together first.

Reply:
Changed and moved up in https://github.com/ros-planning/navigation2/pull/2982/files#diff-95f4c0b773dca7c45bd03829c788a4d6ed3ddf65394702457138ffc35a3355c2R29.